test/built-ins/DataView/prototype/getFloat16/length.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2024 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-dataview.prototype.getfloat16
+description: >
+  DataView.prototype.getFloat16.length is 1.
+features: [Float16Array]
+includes: [propertyHelper.js]
+---*/
+
+assert.sameValue(DataView.prototype.getFloat16.length, 1);
+
+verifyNotEnumerable(DataView.prototype.getFloat16, "length");
+verifyNotWritable(DataView.prototype.getFloat16, "length");
+verifyConfigurable(DataView.prototype.getFloat16, "length");
```

Comment:
We're trying to prefer `verifyProperty` over the individual functions in new tests (command applies to other tests of property descriptors as well)
```suggestion
verifyProperty(DataView.prototype.getFloat16, "length", {
  value: 1,
  enumerable: false,
  writable: false,
  configurable: true,
});
```

Reply:
I just copied these over from `getFloat32`. Would you prefer to use `verifyProperty` even if it means inconsistency between `getFloat16` and all the other dataview methods? Or I can update the other dataview methods as part of this PR and achieve consistency that way, alternatively.

Reply:
I mean, if you feel like doing the extra cleanup work to update the other dataview methods, great! That would be very kind of you and much appreciated. But it's not a requirement as far as I'm concerned.

I do prefer that we don't introduce new tests using APIs that we're trying to phase out, even if it means temporary inconsistency. If we preferred consistency, it'd mean more and more work for the person eventually doing the cleanup.

Reply:
Opened https://github.com/tc39/test262/pull/4023. I'll update whichever PR lands second to match the other.